ui/pages/routes/routes.component.js

```diff
@@ -730,7 +738,10 @@ export default class Routes extends Component {
         <QRHardwarePopover />
         <Modal />
         <Alert visible={this.props.alertOpen} msg={alertMessage} />
-        {!this.hideAppHeader() && <AppHeader location={location} />}
+        {!isConfirmation && !this.hideAppHeader() && (
```

Comment:
Since we already have the `hideAppHeader` to encapsulate these checks, should we include `isConfirmation` in there and maybe put it in it's own method so we can still add the logo below?

Reply:
I implemented the change